harvard-huit/Harvard-Geospatial-Library
# Delete old guests

**Delete Old Guests*
* * *

**JIRA Ticket**: (link)


# What does this Pull Request do?
This adds in an auto-purge of guest users older than 24 hours into our users model to solve the slowdown problem

# How should this be tested?

This is running on dev, but to test on local, first run a query count on your local db for guest users (just to see the number change). Then rebuild the container from this branch, and navigate to the page. Re-query your local db and you will see the number drop. In addition, you shouldn't notice any slowdown on loading pages.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
@enriquediaz
